pkgs/top-level/all-packages.nix

```diff
@@ -617,6 +617,7 @@ in
     clang = clang_9;
     llvm = llvm_9;
     python = python37;
+    pythonPackages = python37Packages;
```

Comment:
You can actually access a `python`'s packages through `python.pkgs`, which would be more elegant in this case than adding a new arg.

Reply:
That's nice, I wasn't aware of that :)